docs/metrics/getting-started-observable-counter/README.md

```diff
@@ -50,14 +50,14 @@ What does the above program do?
 
 The program creates a
 [Meter](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#meter)
-instance named "TestMeter" and then creates a
+instance named "MyMeter" and then creates a
```

Comment:
the sentence below this need to be update as we are now reporting processortime, instead of just any other number.

Reply:
I guess it's probably fine to not mention the details?

Reply:
okay with me